pkgs/development/tools/zigmod/default.nix

```diff
@@ -0,0 +1,159 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchgit
+, zig
+}:
+
+let
+  deps = [
+    {
+      url = "https://gist.github.com/nektro/d468fea84f8217e4c26ee8fbeeea38cc";
+      rev = "0911d65210fe14467fa73afd5b09d781a3995adc";
+      sha256 = "sha256-G62R6rYd2NwMnayyuzZeMLOTb+LYOKeN7ithy8YYTwI=";
+    }
+    {
+      url = "https://github.com/MasterQ32/zig-uri";
+      rev = "e879df3a236869f92298fbe2db3c25e6e84cfd4c";
+      sha256 = "sha256-OUS/5wIg3Le0qjpO1WlvQcI05jhR1k49ABQY+aZSn5E=";
+    }
+    {
+      url = "https://github.com/marlersoft/zigwin32";
```

Comment:
- [zigwin32](https://github.com/marlersoft/zigwin32)
  - comes from [zigwin32gen](https://github.com/marlersoft/zigwin32gen)
    - comes from [win32json](https://github.com/marlersoft/win32json)
      - comes from [win32jsongen](https://github.com/marlersoft/win32jsongen)

This is just ugly.

Reply:
and the best part: we don't need windows